src/install/lockfile/lockfile_json_stringify_for_debugging.zig

```diff
@@ -313,6 +313,18 @@ pub fn jsonStringify(this: *const Lockfile, w: anytype) !void {
                 }
             }
 
+            if (@as(u16, @intFromEnum(pkg.meta.libc)) != Npm.Libc.all_value) {
```

Comment:
```suggestion
            if (pkg.meta.libc != .all) {
```

looks like this change can be made to the others too